tests/provision/virtual.testcloud/test.sh

```diff
@@ -48,6 +48,11 @@ rlJournalStart
             rlAssertGrep "disk\\[0\\].size: set to '11 GB' because of 'disk\\[0\\].size: == 11 GB'" "$run/log.txt"
             rlAssertGrep "final domain memory: 2049000" "$run/log.txt"
             rlAssertGrep "final domain root disk size: 11" "$run/log.txt"
+            rpmdev-vercmp "$(rpm -q --qf '%{V}' testcloud)" 0.11.4 > /dev/null; testcloud_cmp_code=$?
```

Comment:
Is it a good idea to depend on the currently installed `rpm`? What if the test is run from pip installed bits only?

Reply:
Let's disable the new check until fresh `testcloud` is released. Done in bde692d0.

Reply:
I've prepared the change to be merge after `testcloud` is released in #3600.

Reply:
Bumping the require version of testcloud is a more elegant solution, thanks!